packages/@aws-cdk/aws-rds/lib/instance-parameter-group.ts

```diff
@@ -0,0 +1,130 @@
+import { CfnOutput, Construct, IConstruct, Token } from '@aws-cdk/cdk';
+import { Parameters } from './props';
+import { CfnDBParameterGroup } from './rds.generated';
+
+/**
+ * A instance parameter group
+ */
+export interface IInstanceParameterGroup extends IConstruct {
+  /**
+   * Name of this parameter group
+   */
+  readonly parameterGroupName: string;
+
+  /**
+   * Export this parameter group
+   */
+  export(): InstanceParameterGroupImportProps;
+}
+
+/**
+ * Properties to reference a instance parameter group
+ */
+export interface InstanceParameterGroupImportProps {
+  readonly parameterGroupName: string;
+}
+
+/**
+ * Properties for a instance parameter group
+ */
+export interface InstanceParameterGroupProps {
+  /**
+   * Database family of this parameter group
+   */
+  family: string;
+
+  /**
+   * Description for this parameter group
+   */
+  description: string;
+
+  /**
+   * The parameters in this parameter group
+   */
+  parameters?: Parameters;
```

Comment:
Why optional? 

Is there a `@default`?

Reply:
No reason, just followed `ClusterParameterGroup`. I have no own opinion of this and change this to be required.

Reply:
fixed ad3d753